packages/openapi-fetch/test/index.test.ts

```diff
@@ -768,6 +769,33 @@ describe("client", () => {
       expect(getRequestUrl().href).toBe(toAbsoluteURL("/self"));
     });
 
+    it("dispatcher", async () => {
+      const dispatcher = new Agent({
+        connect: {
+          rejectUnauthorized: false,
+        },
+      });
+      let client = createClient<paths>({ baseUrl, dispatcher });
+
+      const { getRequestUrl } = useMockRequestHandler({
+        baseUrl,
+        method: "get",
+        path: "/self",
+        status: 200,
+        body: { message: "OK" },
+      });
+
+      await client.GET("/self");
+
+      // assert baseUrl and path mesh as expected
+      expect(getRequestUrl().href).toBe(toAbsoluteURL("/self"));
```

Comment:
Is this test testing what we think it is? I don’t see how this uses `dispatch` (or maybe I’m missing it?)